internal/cmd/identities_test.go

```diff
@@ -117,9 +117,9 @@ func TestIdentities(t *testing.T) {
 					assert.NilError(t, err)
 
 					respBody := api.CreateIdentityResponse{
-						ID:         uid.New(),
-						Name:       createIdentityReq.Name,
-						ProviderID: providerID,
+						ID:           uid.New(),
+						Name:         createIdentityReq.Name,
+						ProviderName: "infra",
```

Comment:
not sure I understand this. Seems like the only valid value is "infra", and the identity isn't tied to a provider anyway

Reply:
Changed to a "SetOneTimePassword" flag in the request that sets credentials in the Infra provider. Dropped the ProviderName